server/test/index.test.js

```diff
@@ -2,9 +2,10 @@ import { equal, match } from 'node:assert'
 import test from 'node:test'
 import { URL } from 'node:url'
 
-import app from '../index.js'
+process.env.NODE_ENV = 'production'
```

Comment:
There is a better way, than calling `await import('../index.js')`.

Put `process.env.NODE_ENV = 'production'` to `server/test/set-production.js` and import it before all other imports.

Reply:
Done. It looks cleaner 👍